tests/validation/tests/v3_api/test_rbac_2.py

```diff
@@ -0,0 +1,23 @@
+import pytest
+import os
+from .common import \
+    create_kubeconfig, \
+    DATA_SUBDIR, \
```

Comment:
I prefer to have `from .common import x` for each import instead of breaking one line in multiple.

All caps variables go first too in the `.common` group.